app/components/App.jsx

```diff
@@ -28,15 +28,15 @@ export default class App extends React.Component {
 
           <div id='dashboard'>
             <Nav bsStyle='pills' activeKey={this.state.activeKey} onSelect={this.handleSelect}>
-              <NavItem eventKey={'global'      }>  Global scoreboard     </NavItem>
-              <NavItem eventKey={'individual'  }>  Individual scoreboard </NavItem>
-              <NavItem eventKey={'problem-set' }>  Weekly Problem Set    </NavItem>
-              <NavItem eventKey={'all-problems'}>  All Problems          </NavItem>
+              <NavItem eventKey={'global'}> Global scoreboard </NavItem>
+              <NavItem eventKey={'individual'}> Individual scoreboard </NavItem>
+              <NavItem eventKey={'problem-set'}> Weekly Problem Set </NavItem>
+              <NavItem eventKey={'all-problems'}> All Problems </NavItem>
             </Nav>
-            { this.state.activeKey == 'global'       && <GlobalScoreboard /> }
```

Comment:
The spaces around the operators were fine :cry:

Reply:
Ups sorry I remove it by mistake, I thought: this looks nice, it should not be here.

Reply:
I think @rendon was referring to the spaces around the `equals`: 

`{ this.state.activeKey == 'global' && <GlobalScoreboard /> } `. 

Besides that, change looks good to me.

Reply:
Exactly, @Gansito144, you went from `x == y` to `x=y`.

Reply:
I will create a new request in the afternoon. Thank you both.